StratisCore.UI/src/app/wallet/dashboard/dashboard.component.ts

```diff
@@ -112,11 +113,32 @@ export class DashboardComponent implements OnInit, OnDestroy {
     ;
   }
 
+  private setWalletBalance(balanceResponse: any) {
+    if (!this.accountsEnabled) {
+      // TO DO - add account feature instead of using first entry in array
+      this.confirmedBalance = balanceResponse.balances[0].amountConfirmed;
+      this.unconfirmedBalance = balanceResponse.balances[0].amountUnconfirmed;
+      this.spendableBalance = balanceResponse.balances[0].spendableAmount;
+    } else {
+      this.confirmedBalance = balanceResponse.amountConfirmed;
+      this.unconfirmedBalance = balanceResponse.amountUnconfirmed;
+      this.spendableBalance = balanceResponse.spendableAmount;
```

Comment:
`balanceResponse.spendableAmount` does not exist on the response from the WalletController it returns an AddressBalance model see below : or am I missing something?

`public class AddressBalanceModel
    {
        [JsonProperty(PropertyName = "address")]
        public string Address { get; set; }

        [JsonProperty(PropertyName = "coinType")]
        public CoinType CoinType { get; set; }

        [JsonProperty(PropertyName = "amountConfirmed")]
        public Money AmountConfirmed { get; set; }

        [JsonProperty(PropertyName = "amountUnconfirmed")]
        public Money AmountUnconfirmed { get; set; }
    }
}`